Formula/med-file.rb

```diff
@@ -0,0 +1,48 @@
+class MedFile < Formula
+  desc "Modeling and Data Exchange standardized format"
+  homepage "http://www.salome-platform.org"
+  url "http://files.salome-platform.org/Salome/other/med-3.2.0.tar.gz"
+  sha256 "d52e9a1bdd10f31aa154c34a5799b48d4266dc6b4a5ee05a9ceda525f2c6c138"
+
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+  depends_on "swig" => :build
+  depends_on "hdf5"
+  depends_on "python"
+
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/720fedf/med-file/libc%2B%2B_and_python_bindings_fixes.diff"
+    sha256 "4125238ca3623e682b766cf2d34fed17938007ad326ae3b158a9cd427c638054"
+  end
+
+  def install
+    python_prefix=`#{Formula["python"].opt_bin}/python2-config --prefix`.chomp
+    python_include=Dir["#{python_prefix}/include/*"].first
+    python_library=Dir["#{python_prefix}/lib/libpython*.dylib"].first
+
+    system "cmake", ".", "-DMEDFILE_BUILD_PYTHON=ON",
+                         "-DMEDFILE_BUILD_TESTS=OFF",
+                         "-DMEDFILE_INSTALL_DOC=ON",
+                         "-DPYTHON_INCLUDE_DIR=#{python_include}",
```

Comment:
can probably find these on its own

Reply:
I'll try again but previously, I don't think it worked.

Reply:
No joy - need to provide the include path otherwise it can't find Python.h during build.

Reply:
sounds like a :bug: